api/db/drop.js

```diff
@@ -23,27 +23,43 @@ const userArgs = minimist(process.argv.slice(2), {
 
 const dropInterests = async () => {
   console.log('\n', '\x1b[0m', 'Dropping interests collection...');
-  if (await interestCount() !== 0) {
+  if (await getInterestsCount() !== 0) {
     await Interest.collection.drop();
     console.log('\x1b[31m', 'Dropped interests collection');
   } else {
     console.log('\x1b[33m', 'Interests collection is already empty');
   }
+
+  if (await getEventsCount() !== 0) {
+    const drop = await createPrompt(
+      'The events collection is dependent on the interests collection. Drop events collection?'
+    );
+
+    if (drop) await dropEvents(drop);
+  }
+
```

Comment:
Upon further reflection, I think you can get rid of this one, as this question will already be asked if the developer decides to drop the users collection, and if not, they likely won't want to drop the events one either. Feel free to go either way!